Wabbajack/UI/ReactiveMetro.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Windows;
+using MahApps.Metro.Controls;
+using ReactiveUI;
+
+namespace Wabbajack.UI
```

Comment:
Maybe this is for another discussion, but I'm usually not of the opinion that classes' namespaces should match their contained folders.  I feel like this leaders to either excessive namespace definitions... or an unorganized codebase as you try to simplify the namespaces by putting -everything- in the same folder.

Especially in an end-use project like this, I tend to just use one namespace despite using organization folders.  So I'd recommend swapping the namespace to `Wabbajack`?